model/manifest.go

```diff
@@ -130,6 +130,10 @@ type Manifest struct {
 	// A description of what your plugin is and does.
 	Description string `json:"description,omitempty" yaml:"description,omitempty"`
 
+	// A relative file path in the bundle that points to the plugins svg icon.
+	// This should be relative to the root of your bundle and the location of the manifest file.
```

Comment:
```suggestion
	// This should be relative to the root of your bundle and the location of the manifest file. Bitmap image formats are not supported.
```

Reply:
@lieut-data It's good that you are mentioning this. Are we support anything besides svg? I don't reminder.

Reply:
No, we aren't supporting anything but SVG -- primarily because we're embedding the payload in the response from the marketplace, and binary images would be expensive / not scale well / etc.